git_machete/tests/functional/test_machete.py

```diff
@@ -55,7 +55,7 @@ def check_out(self, branch: str) -> "GitRepositorySandbox":
     def commit(self, message: str = "Some commit message.") -> "GitRepositorySandbox":
         f = "%s.txt" % "".join(random.choice(string.ascii_letters) for _ in range(20))
         self.execute(f"touch {f}")
-        self.execute(f"git add {f}")
+        self.execute(f"git add {f} >/dev/null")
```

Comment:
All these changes to the git commands themselves aren't probably needed now that you use `check_output`